dotnet/CoreLib/DataFormats/Image/Tesseract/DependencyInjection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticMemory.DataFormats.Image;
+using Microsoft.SemanticMemory.DataFormats.Image.Tesseract;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.SemanticMemory;
+
+public static partial class MemoryClientBuilderExtensions
+{
+    public static MemoryClientBuilder WithTesseractOCR(this MemoryClientBuilder builder, TesseractConfig config)
+    {
+        builder.Services.AddTesseractOCR(config);
+
+        return builder;
+    }
+}
+
+public static partial class DependencyInjection
+{
+    public static IServiceCollection AddTesseractOCR(this IServiceCollection services, TesseractConfig config)
+    {
+        return services
+            .AddSingleton<TesseractConfig>(config)
+            .AddTransient<IOcrEngine, TesseractOcrEngine>();
```

Comment:
Is it heavy to load the Tesseract engine? I think it needs to read a fairly large data file.

Reply:
Yes, I see what you mean.  Perhaps the default option should be no OCR?  That way the customer has to opt into whichever option. (I liked having an active default, but there's custom steps for both Tesseract and Azure API.)

@dluc - What do you think about default OCR functionality?  (Is null/none with opt-in sufficient?)

Reply:
I ended up removing Tesseract (CC can managed that dependency via custom injection) and verified Azure Forms integration.